src/TabViewPanResponder.js

```diff
@@ -41,8 +41,6 @@ function forHorizontal(props: Props) {
       return gestureState.vx < 0;
     } else if (startDirection < 0) {
       return gestureState.vx > 0;
-    } else {
```

Comment:
@satya164 True nothing will change, but it was an unnecessary check. I will change the check to something like this instead:

```
if (vx < 0) {

} else {

}
```